hipcat/cli.py

```diff
@@ -70,14 +70,19 @@ def base_url(self):
 @click.command(help=__doc__)
 @click.version_option()
 @click.argument('room')
+@click.option('--notification', help='Whether the message should be sent as a notification instead.', is_flag=True)
 @click.option('-m', '--message', help='A message to post. Uses STDIN if not provided.')
 @click.option('-q', '--quote', help='Prefix the message with /quote for formatting.', is_flag=True)
 @click.option('-c', '--code', help='Prefix the message with /code for formatting.', is_flag=True)
-def main(room, message, quote, code):
+@click.option('-s', '--sender', help='A label to be shown in addition to the sender\'s name (required if using --notify).')
+@click.option('--color', help='Background color for message. Valid values: yellow, green, red, purple, gray, random.')
```

Comment:
It would be good to have some validation of the color value so we could tell the user what they did wrong.